style/style.css

```diff
@@ -0,0 +1,118 @@
+/* Mobile */
+
+/* Global */
+*,
+::after,
+::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: arial, sans-serif;
+}
+
+/* Toolbar */
+header {
+  display: flex;
+}
+
+.logo {
+  padding: 20px 20px;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: center;
+  flex: 1;
+  padding: 10px 20px;
+}
+
+nav > div {
+  width: 16px;
+  height: 2px;
+  background: #000001;
+  margin: 2px;
+  border-radius: 50px;
+}
+
+/* Headline */
+main {
+  height: 130vh;
+  padding: 20px 0;
+}
+
+#headline {
+  background-image: url("/images/bg1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: scroll;
+  height: inherit;
+  padding: 0 20px;
+  width: 100%;
+}
```

Comment:
- [ ] You've done a good job so far, but as required by the previous reviewer kindly get your background image looking closer to the figma design. The intention is to get it as close as possible. Remember the UI is already designed, it's your job to follow it. That's how it is in the real world. A UI designer is hired to do the design that has good UX etc. and the Developer is hired to follow the design (color theme, fonts etc.) as closely as possible.

##### Figma Desgin
![image](https://user-images.githubusercontent.com/29071984/176718763-94ed30b9-97f6-4ffb-b1f8-21e8ee289b84.png)

##### Your design
![image](https://user-images.githubusercontent.com/29071984/176719234-69745bf6-adbe-4daa-8db3-2a9b5c8a7a09.png)

Reply:
@Bateyjosue Yes that's totally fine. If your HTML is structured exactly as how the Figma template is, it should work fine

Reply:
@Bateyjosue I personally didn't choose this template for my portfolio project.